fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -2933,22 +2933,20 @@ Either the payer DFSP or the payee DFSP may request currency conversion. If the
 
 ##### 6.12.2.1 Conversion Quote Expiry Details
 
-In the same way as with an ordinary quotation (see [Section 6.5.2.1](#6521-quote-expiry-details) above), the requesting DFSP should include an expiry time for the quotation.
+In the same way as with an ordinary quotation (see [Section 6.5.2.1](#6521-quote-expiry-details) above), the FSP requesting the quotation should include an expiry time for the quotation.
```

Comment:
Suggesting to remove "above", as using "above" might increase the required maintenance of the document (a move might place the section below instead)..